generators/kubernetes/templates/_README-KUBERNETES.md

```diff
@@ -0,0 +1,154 @@
+# JHipster generated kubernetes configuration
```

Comment:
Do you want to add a statement that this generator needs to be run from a sub-folder to have a clean generation. If we run from the base folder it prompts for replacing the app generated README.md file. I know first time if we run from the base will miss the message, but at least from the subsequent runs it can be avoided.